packages/flutter_tools/test/general.shard/windows/windows_workflow_test.dart

```diff
@@ -3,51 +3,60 @@
 // found in the LICENSE file.
 
 import 'package:flutter_tools/src/base/platform.dart';
-import 'package:flutter_tools/src/features.dart';
 import 'package:flutter_tools/src/windows/windows_workflow.dart';
 
 import '../../src/common.dart';
-import '../../src/context.dart';
 import '../../src/testbed.dart';
 
 void main() {
-  Testbed testbed;
-  FakePlatform windows;
-  FakePlatform notWindows;
-
-  setUp(() {
-    windows = FakePlatform(operatingSystem: 'windows');
-    notWindows = FakePlatform(operatingSystem: 'linux');
-    testbed = Testbed(
-      overrides: <Type, Generator>{
-        Platform: () => windows,
-        FeatureFlags: () => TestFeatureFlags(isWindowsEnabled: true),
-      },
+  final FakePlatform windows = FakePlatform(operatingSystem: 'windows');
```

Comment:
And we can safely assume these never get mutated, leaking state between tests?

Reply:
There are some contrived circumstances where you can mutate the environment field, but only if that is set to a non-const value (it defaults to an empty const map, which is unmodifiable)